gatsby-app/src/components/elements/metaTags.js

```diff
@@ -0,0 +1,54 @@
+import logo from '../../assets/images/favicon.png'
+
+const metaTags = [
+  {
+    name: 'description',
+    content: 'UP - Easy to use private file hosting',
+  },
+  {
+    name: 'charset',
+    content: 'utf-8',
+  },
+  {
+    name: 'keywords',
+    content: 'file, hosting, up',
+  },
+  {
+    name: 'subject',
+    content: 'Private file hosting',
+  },
+  {
+    name: 'copyright',
+    content: 'Starchasers',
+  },
+  {
+    name: 'language',
+    content: 'en-US',
+  },
+  {
+    name: 'robots',
+    content: 'index,follow',
+  },
+  {
+    name: 'author',
+    content: 'Starchasers, kuboczoch@gmail.com',
```

Comment:
```suggestion
    content: 'Starchasers',
```
It's not necessary, should be an organization email.